Swift Chat Demo 2/LoginRegistrationViewController.swift

```diff
@@ -49,6 +50,13 @@ class LoginRegistrationViewController: UIViewController {
                 SVProgressHUD.dismiss(withDelay: 3)
                 return
             }
+            let center = UNUserNotificationCenter.current()
+            center.requestAuthorization(options:[.badge, .alert, .sound]) { (granted, error) in
```

Comment:
I think we don't need `center.requestAuthorization` if we will always `registerForRemoteNotifications` again?